src/less/components.less

```diff
@@ -36,7 +36,9 @@
 @import 'components/popover/index';
 @import 'components/radio/index';
 @import 'components/search/index';
-@import 'components/table/index';
+@import 'components/tabset/index';
+@import 'components/text-input/index';
+@import 'components/textarea/index';
```

Comment:
Revert the addition of these lines. You've got Tabset, Text Input, and Textarea imported twice (once in Lines 39-41, another in Lines 42-44)